settings/aboutpage.md

```diff
@@ -14,43 +14,119 @@ The about page allows to configure some client related configuration
+- Links to the official openHAB Website, the openHAB Documentation and the Community Forum
 
 ## Technical Information
 
-- Configuration folder
-- User data folder
-- Logs folder
-- Operating System
-- Java Runtime
-  - View Details
+This section is collapsed by default and can be opened by clicking on it which then reveals the
+
+- Configuration folder on the server
+- User data folder on the server
+- Logs folder on the server
+- Operating System openHAB is running on
+- Java Runtime openHAB is running on
+- number of processors and memory
+
+Clicking on View Details opens up a popup window that contains very detailed information like
+
+- Runtime information
+- Locale (language)
+- System information including memory information
+- A list of all installed bindings
+- Client information including device information, client settings (see also the below section), screen information, theme, user agent information.
+
+The content can be easily copied with the copy button on the top to provide it as additional information when asking questions in the community.
 
 ## Appearance (for this device only)
 
-- Theme
-![Theme](theme-selection.png)
+This section allows to have different appearance settings _per_ device.
+
+- **Theme**: Selects the theme for that device
   - Auto: tries to detect the client type
   - Android
   - iOS
   - Destktop
-- Dark mode setting
-![Dark-Mode](dark-mode.png)
-  - Auto: **TODO**
+
+![theme-selection.png](images/theme-selection.png)
+
+- **Dark mode** setting
+  - Auto: uses the default mode
   - Light-Mode
   - Dark-Mode
-- Navigation bars style
-![Navigation bars style](navbar-style.png)
+
+![dark-mode.png](images/dark-mode.png)
+
+- **Navigation bars style**: inverts the colors in the navigation bar
+
+![navbar-style.png](images/navbar-style.png)
 
 # Miscellaneous settings
 
-- Simple navigation bar on home page: **TODO** What is the navigation bar
-- Standard home page background color: **TODO**
-- Disable card expansion animations: Disables the animations when expanding a widget card
-- Disable page transition animations: Disables the transitions when switching from one page to the other
-- Enable Web Audio sink support
-  - **TODO** : see [https://github.com/openhab/openhab-webui/pull/1422](https://github.com/openhab/openhab-webui/pull/1422)
+- **Simple navigation bar on home page**: When switching it on, it means the navigation bar will look the same even whether you're at the top of the page or not - you won't have the alternate look with the date.
+The first image depicts the normal setting (off), the second image shows it switched off
+
+![simple-nav-off.png](images/simple-nav-off.png) ![simple-nav-on.png](images/simple-nav-on.png)
+
+- **Standard home page background color**: This only relates to the light color scheme. The background color will be the same as the regular pages you can show from e.g. the sidebar, which is a kind of gray instead of white.
```

Comment:
```suggestion
- **Standard home page background color**: This only relates to the light color scheme.
The background color will be the same as the regular pages you can show from e.g. the sidebar, which is a kind of gray instead of white.
```